src/game.js

```diff
@@ -0,0 +1,33 @@
+class Game {
+	constructor() {
+		this.id = 0;
+		this.leftAttempts = 5;
+	}
+
+	static create() {
+		return new Promise((resolve, reject) => {
+		var json = {
+			id: 1,
+        	hint: '_ _ _ _ A',
+        	leftAttempts: 5
+		}
+    	if(Object.keys(json).length != 0) {
+			return resolve(json)		
+    	}
+    	else {
+    		return reject()
+    	}
+		})
+	}	
+
+	static save(data) {
+		return new Promise((resolve,reject) => {
+			var map = new Map();
```

Comment:
this won't persist any game because this map will be deleted in every call to the save method